File: bulk/java/InsecureCookie01.java
// CWE-614: Cookie without Secure and HttpOnly flags
import javax.servlet.http.*;
public class InsecureCookie01 {
    public void setSession(HttpServletResponse resp, String token) {
        Cookie c = new Cookie("session", token);
        // missing: c.setSecure(true); c.setHttpOnly(true);
        resp.addCookie(c);
    }
}

File: bulk/csharp/InsecureCookie01.cs
// CWE-614: Session cookie without Secure/HttpOnly
using Microsoft.AspNetCore.Http;
public class InsecureCookie01 {
    public void SetSession(HttpResponse response, string token) {
        response.Cookies.Append("session", token); // missing Secure=true, HttpOnly=true
    }
}
